tests/catalog/test_context.py

```diff
@@ -0,0 +1,62 @@
+import unittest
```

Comment:
in this tests we don't document=test contexts functionality, but it's inner (and temporary because of it) realization.
I think, that tests should be based on in/out interfaces, but not on inner calls.

But maybe it's about taste and my school

Reply:
@duker33 I agree with the point, but unfortunately we still haven't `Tag` fixtures, so i have to either use mocked objects or create tags at a place of use. I don't like both of these approaches, but right now mocked objects looking better. I will create a todo to use black box testing only.